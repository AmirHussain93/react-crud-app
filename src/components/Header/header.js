import React from 'react';
import { Button } from 'react-bootstrap';

import './header.scss';


function Header() {
    return (
        <div className="header-section">
            <div className="inner-box">
                <Button className="header-items">Employees</Button>
                <Button className="header-items">Add Employee</Button>
            </div>
        </div>
    )
}

export default Header