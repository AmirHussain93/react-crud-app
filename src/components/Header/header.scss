@import "~bootstrap/scss/bootstrap";

.header-section {
    width: 100%;
    background-color: #000;

    .inner-box {
        padding: 5px;;
    }

    .header-items {
        margin-left: 10px;
    }
}