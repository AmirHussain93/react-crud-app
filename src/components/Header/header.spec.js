import React from 'react';
import { shallow } from 'enzyme';
import Header from './header';

describe('Header', () => {

    let wrapper;

    beforeEach(() => wrapper = shallow(<Header />))

    it('should render correctly', () => expect(wrapper).toMatchSnapshot());

    it('it should render 2 </div>`/s', () => {
        expect(wrapper.find('div').length).toEqual(2);
    })

    it('it should render 2 </Button> `/s', () => {
        expect(wrapper.find('Button').length).toEqual(2);
    })
})