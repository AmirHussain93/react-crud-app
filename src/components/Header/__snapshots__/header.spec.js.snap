// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<div
  className="header-section"
>
  <div
    className="inner-box"
  >
    <Button
      active={false}
      className="header-items"
      disabled={false}
      variant="primary"
    >
      Employees
    </Button>
    <Button
      active={false}
      className="header-items"
      disabled={false}
      variant="primary"
    >
      Add Employee
    </Button>
  </div>
</div>
`;
